docs/UserGuide.md

```diff
@@ -232,7 +234,50 @@ to `91234567` and `johndoe@example.com` respectively.
 ### Managing Appointments
 [To be filled up]
 ### Managing Medical Conditions
-[To be filled up]
+
+#### Adding Medical Conditions : `addMedCon`
+
+Adds medical condition to an existing patient in MediBase3.
+
+Format: `addMedCon i/NRIC c/CONDITION`
+
+{: .alert .alert-info}
+> :information_source: **Note:**
+>
+> * Adds medical condition to the patient with the specified `NRIC` in MediBase3.
+> * You can add a medical condition to a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after medical condition has been added.
+> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+
+Example: 
+* `addMedCon i/S1234567A c/High Blood Pressure` will add medical condition `High Blood Pressure` to patient with NRIC `S1234567A`.   
+
+{: .alert .alert-success}
+> :bulb: **Tip:**
+>
+> User can add more than 1 Medical Condition through using `c/CONDITION` multiple times
+
+#### Deleting Medical Conditions : `delMedCon`
+
+Deletes Medical Condition from an existing patient in MediBase3.
```

Comment:
I think can keep 'medical condition' in lowercase to remain consistent with the addMedCon command

Reply:
Good catch! Will make "medical condition" in addMedCon command to be upper caps as well to ensure consistency, viewing it as an object, hence the upper caps.